keyboards/aeboards/aegis/keymaps/via/rules.mk

```diff
@@ -0,0 +1,68 @@
+# project specific files
```

Comment:
@Wilba6582 Is it going to be customary for `via` keymaps to include the entire contents of the keyboard's `rules.mk` file?

Reply:
In theory, only this is necessary, as far as I'm aware:
```make
BOOTMAGIC_ENABLE = no      # Virtual DIP switch configuration(+1000)
MOUSEKEY_ENABLE = no       # Mouse keys(+4700)
EXTRAKEY_ENABLE = yes       # Audio control and System control(+450)
CONSOLE_ENABLE = no        # Console for debug(+400)
COMMAND_ENABLE = no        # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
UNICODE_ENABLE = no         # Unicode
BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
AUDIO_ENABLE = no           # Audio output on port C6
FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches

RAW_ENABLE = yes
DYNAMIC_KEYMAP_ENABLE = yes 

SRC =	keyboards/wilba_tech/wt_main.c

Reply:
$reasons

Reply:
Is there a specific reason for it, though? 

It looks to be identical to the main `rules.mk` for the keyboard, so ... it's redundant.

Reply:
The specific reason is the same as the last time you asked me this exact same question:

> It's a placeholder for me coming back to these and refactoring it down to the things that need to be explicitly set for VIA compatibility (esp. feature defines), not that it wouldn't work now as just the delta with the parent.

Also, the "via" keymap will build the same regardless of (most) end user edits in the parent `rules.mk`

Just because something is redundant doesn't automatically make it bad.

For now, while we're in this intermediate stage of VIA support being in a state of flux, can we be a little less pedantic about my commits?

Reply:
Okay. Thanks